addons/mail/static/src/models/thread/thread.js

```diff
@@ -1840,6 +1860,11 @@ function factory(dependencies) {
         isTemporary: attr({
             default: false,
         }),
+        /**
+         * Date object describes when the last meaningful action is done in a thread,
+         * e.g. pin or send/receive messages.
+         */
+        lastMeaningfulActionTime: attr(),
```

Comment:
I don't like the name of this field. Notably due to being far too vague.
Would prefer something simpler like `lastChangedDate`.

I get "changed" is not that much specific, but at least it's far clearer that it's about something that is perceived as changed.

It's a bit misleading in that it may look like a field that is updated only from intrinsic changes to the thread, but it's alright to keep wording simple and clearly explain what is meant by a thread being "changed".

IMO "changed" is a log resulting to transition from one state to another, but it's wrong to assume it's only about intrinsic state. We are used to tell a file or an object is "changed", therefore we are used to consider intrinsic changes, but that reductive to exclude extrinsic stuffs in some applications.

Reply:
There are a lot of changes that should not change the value of this field. "changed" is way too generic, there is already `write_date` field for that and it's useless in our case.